_docs/services/relational-database.md

```diff
@@ -117,38 +117,44 @@ In short, `cf bind-service` will provide a `DATABASE_URL` environment variable f
-requires manually downloading the tool(s) needed to access the database.
+- Your organization name
+- The space you are working within
+- The name of the application(s) connected to the database service you need a restoration performed on
 
-### cg-migrate-db plugin
-You can access the data in your database via the `cg-migrate-db`
-plugin. See the [repository](https://github.com/18F/cg-migrate-db)
-for instructions on how to install the plugin, backup data, import data,
-download a local copy of the data, and upload a local copy of the data.
+We'll confirm this information and remind you that a restoration may result in a brief period of downtime with database connectivity.  Once we receive confirmation from you to proceed, we'll perform the restore, which results in a new DB instance being created in AWS RDS.  cloud.gov support will take care of renaming the new instance and configuring it with the same VPC and security group as the old instance in AWS so that it can still be found by your bound application(s) once the restoration is complete.
 
-### Manually access a database
+When the restore process is completely finished, we'll notify you and ask you to confirm that your application(s) is still functioning properly and that the data is properly restored.  We'll also coordinate with you to determine when it would be appropriate to remove the old instance, particularly if it is needed for something such as a security audit or forensic analysis.
 
-#### Using cf ssh
+You can also create manual backups using the [export process](#exporting-a-database) described below. In general, you are responsible for making sure that your backup procedures are adequate for your needs; see CP-9 in the cloud.gov SSP.
 
-To access a service database, use the [cf-service-connect plugin](https://github.com/18F/cf-service-connect#readme).
+## Access the data in the database
 
-### Export
+To access a service database, use the [cf-service-connect plugin](https://github.com/18F/cf-service-connect#readme) and the corresponding CLI (command line interface) tools for the database service you are using.
 
-#### Create backup
+### Exporting a database
 
 The instructions below are for PostgreSQL, but should be similar for MySQL or others.
 
-First, connect to an instance using the [cf-service-connect plugin](https://github.com/18F/cf-service-connect#readme):
+First, open a terminal and connect to an instance using the [cf-service-connect plugin](https://github.com/cloud-gov/cf-service-connect#usage) to create a SSH tunnel:
 
 ```sh
-$ cf connect-to-service --no-client ${APP_NAME} ${SERVICE_NAME}
+# Find the aws-rds service you are targeting in the services list and make note of the service's name
+$ cf services
+
+# Check to see if a SERVICE_CONNECT service key already exists for the service.
+$ cf service-keys ${SERVICE_NAME}
+
+# If a key exists, remove it first before starting.
+$ cf delete-service-key ${SERVICE_NAME} SERVICE_CONNECT
```

Comment:
Just curious why this step is needed, to delete an existing `SERVICE_CONNECT` service key. Won't the `connect-to-service` plugin just reuse an existing key if it exists, or create. a new one automatically if it doesn't?

Reply:
Theoretically `connect-to-service` should reuse the existing key and I believe it does, but there's a situation where that key becomes "stale" so to speak.  I think it's if an app is restaged/redeployed, because the plugin requires an app that's bound to the service instance.  If the app isn't the same app instance as what was used before when first creating the connection, `connect-to-service` can't reconcile that and it won't create a new one automatically.

I'll investigate a bit further to see if that's still the case or if my memory isn't serving me correct here!

Reply:
I was slightly mistaken, it isn't the app instance that causes the existing key to become stale, it's deleting the original service the key was bound to and binding a new instance in its place with the same service name.  In other words, if you follow this flow (in its simplest form):

- Connect to existing DB service named `mydb` with the plugin
- Unbind and delete the existing DB service once you're done with it
- Create and bind a new DB service named `mydb` again
- Try to connect to the new DB instance

The reason why this will fail is because the service key will still have the credentials and information of the original service information.  When you provision a new service, a whole new identifier is given and the credentials will be different (this is why we have to be careful in the operator runbook to match everything up when restoring an instance).

For the purposes of this customer documentation as it is written here, you're right, this should not be needed and if anything, only mentioned as a troubleshooting step if necessary.

Thanks for catching and asking, @mheadd!